src/gt4py/next/__init__.py

```diff
@@ -39,6 +39,11 @@
     index_field,
     np_as_located_field,
 )
+from .program_processors.runners.gtfn import (
+    run_gtfn_cached as gtfn_cpu,
+    run_gtfn_gpu_cached as gtfn_gpu,
```

Comment:
On a second thought, I think we should keep the very explicit names, because people (at this stage at least) should know what they are using and being able to communicate to us. If we go for a default and switch it later, it might add unnecessary confusion.